### DISCUSSION TITLE
🚀 Feature request - Structure - can the "view page" button open in a new window

### DISCUSSION THREAD

**Discussion by agun** [ExpressionEngine/ExpressionEngine#3834] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
In the control panel, when you are on the Structure main page (showing the full structure), when you hover over a page, you can see 3 buttons (assuming you have full access). One of the buttons is "view page". 

It would be great if this button opened the page in a new tab/window (ie add `target="_blank"` to the button link). Currently the button opens the page in the same window ... which then loses your place in the control panel.

**Describe the solution you'd like**
Add `target="_blank"` to the Structure "View Page" button.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by swierczek** [comment]
It feels a little prescriptive to force this to open in a new tab, when not all users would necessarily prefer that, and middle clicking or `ctrl + click` on Windows or `cmd + click` on Mac opens links in new tabs per native browser functionality.

Edit: just kidding, I didn't realize that this opens the front-end page. I don't have a strong opinion in that case. 👍

**Reply by agun** [reply]
Thanks @swierczek  ... I would agree if it was opening a page in the control panel ... but it opens the actual front-end page, which feels like it should open like an 'external' link (in a new window).

**Reply by swierczek** [reply]
In fact, clicking the site name in the top left corner opens the homepage in a new tab already. So that would be a consistent experience.

**Comment by intoeetive** [comment]
Makes sense

https://github.com/ExpressionEngine/ExpressionEngine/pull/3836
